install/update.php

```diff
@@ -8683,6 +8683,9 @@ function migrateTablesFromFusinvDeploy($migration)
          //"deployorders fixer : final order structure for ID ".$order_config['id']."\n" .
        //   json_encode($json_order,JSON_PRETTY_PRINT) ."\n"
        //);
+        $migration->addField('glpi_plugin_glpiinventory_taskjobs', 'restrict_to_task_entity', 'bool', ['value'   => '1']);
```

Comment:
I guess it should then be removed from `update.native.php` file, no?

Reply:
Since last commit, this is no longer necessary